nemo_aligner/algorithms/ppo.py

```diff
@@ -348,10 +348,16 @@ def run_training(self, dataloader_iter):
         return loss_mean, metrics
 
     def fit(self):
-        if self.cfg.max_epochs is not None and self.cfg.max_epochs > 1:
-            # because we need to figure out a nice way to reset the shuffling on our dataset
-            # otherwise epoch > 1 will loop over the dataset in the same order
-            raise ValueError("epoch > 1 is not supported")
+        if (not isinstance(self.train_dataloader.batch_sampler, MegatronPretrainingRandomBatchSampler)) and (
```

Comment:
Just a note that the import of `MegatronPretrainingRandomBatchSampler` is missing.

Reply:
Since we have decided not to use `MegatronPretrainingRandomBatchSampler` for PPO, I will revert this file back to how it was before.